src/applications/edu-benefits/components/OptOutWizard.jsx

```diff
@@ -41,6 +49,7 @@ export default class OptOutWizard extends React.Component {
           <p><strong>Please note:</strong> If you opt out and then change your mind, you’ll need to call the Education Call Center at <a className="help-phone-number-link" href="tel:+1-888-442-4551">1-888-442-4551</a> to opt back in.</p>
           <div>
             <a
+              onClick={this.confirmOptOut}
```

Comment:
Does this event fire before navigation?

Reply:
Technically I believe it does. Is that a concern?

Reply:
I'm concerned the async call to GA will not fire before browser navigation is initiated but it's been a while since I've worked on something like this.

Reply:
so then the URLs that gets reported maybe won't match what it was when the button was actually clicked?

Reply:
I just dunno which one of the things happens first: `onClick` or `href` navigation. The model in my head is that they both get queued up and `onClick` queues a fetch that is sequenced after the page navigation.

Reply:
It certainly appears that the onClick fires first. And I think that makes sense. In a standard, non-SPA, if the location was updated first, via the `href`, that would trigger a page refresh, effectively killing all of the JS code on the first page. Thus the `onClick` would never fire because the JS no longer exists.

Reply:
okay we really need to talk with @nedierecel and/or Tony to get more info/background on why we seem to be tracking both clicks and the page views that those clicks trigger.

Reply:
@nedierecel could you please confirm that the modal button needs to be tracked, as it is actually a link?